rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
 
-4. In the event of a casual vacancy in any office referred to in subrule (1), the committee may appoint one of its members to the vacant office and the member appointed may continue in office up to and including the conclusion of the annual general meeting next following the date of the appointment. 
+#### 56 Filling casual vacancies
 
-## 23. Ordinary members of the committee
+1. The Committee may appoint an eligible member of the Association to fill a position on the Committee that—
+  1. has become vacant under rule 55; or
+  2. was not filled by election at the last annual general meeting.
 
-1. Subject to these Rules, each ordinary member of the committee shall hold office until the annual or special general meeting roughly 12 months after the date of election but is eligible for re-election. 
+2. If the position of Secretary becomes vacant, the Committee must appoint a member to the position within 14 days after the vacancy arises.
 
-2. In the event of a casual vacancy occurring in the office of an ordinary member of the committee, the committee may appoint a member of the Association to fill the vacancy and the member appointed shall hold office, subject to these Rules, until the conclusion of the annual general meeting next following the date of the appointment. 
+3. Rule 56 also applies to any committee member appointed by the Committee to fill a vacant position under subrule 1 or 2.
 
-## 24. Election of officers and ordinary committee members
+4. The Committee may continue to act despite any vacancy in its membership.
 
-1. Nominations of candidates for election as officers of the Association or as ordinary members of the committee must be—
-  1. made in writing, signed by two members of the Association and accompanied by the written consent of the candidate (which may be endorsed on the form of nomination); or 
-  2. made in person with two endorsements at a general meeting. 
+### Division 4—Meetings of Committee
 
-2. A candidate may only be nominated for one office, or as an ordinary member of the  committee, prior to elections at general meetings. 
+#### 57 Meetings of Committee
 
-3. If insufficient nominations are received to fill all vacancies on the committee, the  candidates nominated shall be deemed to be elected and further nominations may be  received at annual general meeting. 
+1. The Committee meets once a month through the use of web video technology.
 
-4. If the number of nominations received is equal to the number of vacancies to be filled,  the persons nominated shall be deemed to be elected. 
+2. Special committee meetings may be convened by the President or by any 4 members of the Committee.
+  
+#### 58 Notice of meetings
 
-5. If the number of nominations exceeds the number of vacancies to be filled, a ballot  must be held. 
+1. If a special committee meeting is convened, the Secretary must give Committee Members notice by way of email which must include the general nature of the business to be conducted.
```

Comment:
In the original document, the subrules of Rule 58 are numbered 4 and 5. Subrules 1, 2 and 3 appear to be missing, or were removed. I have renumbered these subroles 1 and 2 (since Markdown doesn’t support lists that don’t start at 1), but someone should confirm that the “missing” subroles were indeed omitted intentionally.